### PR TITLE
{math}[CrayGNU/2016.03] NTL v9.8.1

### PR DESCRIPTION
**Pull request by gppezzi**
(created using `eb --new-pr`)

edit: requires #3181

### CODE REVIEW

File: easybuild/easyconfigs/n/NTL/NTL-9.8.1-CrayGNU-2016.03.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+toolchain = {'name': 'CrayGNU', 'version': '2016.03'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.shoup.net/ntl/']
+
+dependencies = [('GMP', '6.1.0')]
+
+prefix_opt = 'PREFIX='
+
+configopts = 'CXX=$CXX GMP_PREFIX=$EBROOTGMP'
```
Comment (boegel) [inline easybuild/easyconfigs/n/NTL/NTL-9.8.1-CrayGNU-2016.03.eb]: @gppezzi to be on the safe side, use `CXX="$CXX"` (`$CXX *may* contain spaces)

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
dora102 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/5985a563af14ef9aae22e1608bd303e1 for a full test report.

**Comment by boegel** [comment]
lgtm :+1:

**Comment by boegel** [comment]
Going in, thanks @gppezzi!

**Pull request merged**